source/features/fix-no-pr-search.tsx

```diff
@@ -5,11 +5,22 @@ import features from '../feature-manager.js';
 import SearchQuery from '../github-helpers/search-query.js';
 
 function redirectToIssues(event: DelegateEvent<Event, HTMLFormElement>): void {
+	event.preventDefault();
+
 	const form = event.delegateTarget;
-	const searchQuery = new SearchQuery(form.elements['js-issues-search'].value);
+	const searchValue = form.elements['js-issues-search'].value;
```

Comment:
If this is missing, just return early, before calling preventDefault. No need for `redirect`. I don’t think the previous feature structure need to be changed this much. 

In what case is it missing by the way?

Reply:
if `url === ''`, it will show a error like 👇

<img width="571" alt="image" src="https://github.com/user-attachments/assets/cf474623-5c32-432b-be41-b85934aafbaa">

<img width="500" alt="image" src="https://github.com/user-attachments/assets/894e6806-2f4b-4b45-a856-1306f5f3d5e7">

Reply:
I'm gonna guess that this feature never worked, SearchQuery accepts a full URL, it shouldn't take random strings from this input.

I think the correct usage is:

```js
const query = SearchQuery.from(location)
query.set(form.elements['js-issues-search'].value);
```

Reply:
done!